fontc/src/workload.rs

```diff
@@ -303,6 +303,7 @@ impl<'a> Workload<'a> {
                     nth_wave += 1;
                 }
 
+                let mut job_count = launchable.len();
```

Comment:
Can you add a comment this is the count of runnable jobs? - at a first glance I thought this was Very Wrong and then I realized it was effectively filtering the !run ones

Reply:
or I can just rename the variable to runnable_count

Reply:
Done in https://github.com/googlefonts/fontc/pull/600/commits/8fff172087baba11d4c8c56d0f97f5bf73f26a12 (squashed)